di-ipv-orchestrator-stub/src/main/java/uk/gov/di/ipv/stub/orc/config/OrchestratorConfig.java

```diff
@@ -9,8 +9,6 @@ public class OrchestratorConfig {
                     "IPV_BACKCHANNEL_ENDPOINT", "https://api.identity.build.account.gov.uk/");
     public static final String IPV_BACKCHANNEL_TOKEN_PATH =
             getConfigValue("IPV_BACKCHANNEL_TOKEN_PATH", "/dev/token");
-    public static final String IPV_BACKCHANNEL_USER_IDENTITY_PATH =
-            getConfigValue("IPV_BACKCHANNEL_USER_IDENTITY_PATH", "/dev/user-identity");
```

Comment:
Do you think it would be preferable to add a new config value for `IPV_BACKCHANNEL_REVERIFICATION_PATH`?

Is there any chance we'll need different paths in different environments? I'm curious about the `/dev/` prefix in the default values, which you've lost.

Reply:
We currently set `IPV_BACKCHANNEL_USER_IDENTITY_PATH` as "user-details" in each env. In addition, I think allowing different paths per env is unnecessary for a stub since `IPV_BACKCHANNEL_ENDPOINT` distinguishes the endpoints between envs. 

The `dev` config (in core-common-infra) used to have `IPV_BACKCHANNEL_ENDPOINT` be "https://api.identity.build.account.gov.uk/", as it is for `build`. This would mean they'd have to specify different paths to not overlap. Now, it's set as "https://api.cf-dev.dev.identity.account.gov.uk/" (Vam made the change in June 2023)

Implementation is also a bit complicated since each of these config values needs to be set in `stub-common-infra` (`build`) and `core-common-infra` (`dev01` & `02`).

Reply:
Ok sure, happy for us to bake in those paths. (You could do the same with token as well for consistency, but not too fussed)

That dev domain appears broken, so I think it must work differently in dev anyway, but we can worry about that another day.